Add Uptime Monitoring Documentation (#10810)
getsentry/sentry-docs: docs/product/alerts/uptime-monitoring/index.mdx

---
title: Uptime Monitoring
sidebar_order: 50
description: "Learn how to help maintain uptime for your web services by monitoring relevant URLs with Sentry's Uptime Monitoring."
---

<Include name="feature-stage-alpha-uptime.mdx" />

Sentry's Uptime Monitoring lets you monitor the availability and reliability of your web services effortlessly.
In the current version, uptime is [automatically configured](/product/alerts/uptime-monitoring/automatic-detection/) as a new alert for only the most relevant URL detected in your organization. In future updates, you'll have the flexibility to add and monitor additional URLs.

## Uptime Check Criteria

Our uptime monitoring system verifies the availability of your URLs
by performing GET requests at regular 5-minute intervals.
For a URL to be considered up and running, the response must meet the following criteria:

1. **Successful Response (2xx Status Codes):**
The URL must return an HTTP status code in the 200–299 range, indicating a successful request.

> 2. **Automatic Handling of Redirects (3xx Status Codes):** URLs returning an HTTP status code in the 300–399, indicating a redirect, will trigger Sentry to automatically follow the redirect and verify that the final destination URL returns a successful response. This ensures that redirects won't falsely trigger downtime alerts.

Reviewer: Wording feels a little weird here
```suggestion
2. **Automatic Handling of Redirects (3xx Status Codes):** Sentry will follow redirects for URLs returning an HTTP status code in the 300–399 range and verify that the final destination URL returns a successful response. This ensures that redirects won't falsely trigger downtime alerts.
```

> 3. **Timeout Setting:** Each request has a timeout threshold of 10 seconds.

Reviewer: It's actually 5 seconds right now isn't it @wedamija ?

Reviewer 2: It's 10 seconds right now
